web/templates/stream_settings/confirm_stream_privacy_change_modal.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="confirm-stream-privacy-modal">
+    <div class="new-style">
+        <p class="confirm-stream-privacy-info">
+            {{t "Changing the privacy of this stream will make the entire message history accessible according to the new
+              privacy configuration."}}
```

Comment:
This line-wrapping seems awkward; I'd prefer to just have the whole string on one line or to wrap at a cleaner midpoint.